### PR TITLE
(SIMP-2473) CentOS 1611 build

### PR DESCRIPTION
**Pull request by jeefberkey**
This is an update to build assets for creating a SIMP ISO based on the CentOS 1611 image. It also finishes the move to Puppet 4 and AIO packaging. 

However, it is not ready for merging.

### CODE REVIEW

**Comment by jeefberkey** [comment]
@jeannegreulich what exactly was the issue?

File: Puppetfile.tracking [hunk 1]
```diff
@@ -324,11 +324,11 @@ mod 'simp-selinux',
   :ref => 'master'
 
 mod 'simp-simp',
-  :git => 'https://github.com/simp/pupmod-simp-simp',
-  :ref => 'master'
+  :git => 'https://github.com/trevor-vaughan/pupmod-simp-simp',
+  :ref => 'SIMP-2469'
```
Comment (trevor-vaughan) [inline Puppetfile.tracking]: Why is this here?

File: Puppetfile.tracking [hunk 2]
```diff
@@ -21,119 +21,119 @@ moduledir 'src/puppet/modules'
...
 mod 'puppetlabs-apache',
   :git => 'https://github.com/simp/puppetlabs-apache',
-  :ref => 'simp-master'
+  :ref => 'master'
 
 mod 'puppetlabs-concat',
-  :git => 'https://github.com/simp/puppetlabs-concat',
-  :ref => 'simp-master'
+  :git => 'https://github.com/jeefberkey/puppetlabs-concat',
+  :ref => 'MODULES-4264'
```
Comment (trevor-vaughan) [inline Puppetfile.tracking]: Add a comment explaining this

File: Puppetfile.tracking [hunk 3]
```diff
@@ -8,8 +8,8 @@ mod 'simp-doc',
   :ref => 'master'
 
 mod 'simp-rsync',
-  :git => 'https://github.com/simp/simp-rsync',
-  :ref => 'master'
+  :git => 'https://github.com/trevor-vaughan/simp-rsync',
+  :ref => 'SIMP-2469'
```
Comment (trevor-vaughan) [inline Puppetfile.tracking]: Why is this here?

File: Puppetfile.tracking (hunk 1 above)
Comment (trevor-vaughan) [inline Puppetfile.tracking]: Why is this here?

File: build/distributions/CentOS/7/x86_64/DVD/ks/dvd/min.cfg [hunk 4]
```diff
@@ -103,7 +103,6 @@ for x in `ls -d /lib/modules/*`; do
 done
 
 /sbin/chkconfig --add network;
-/sbin/chkconfig --level 345 nscd on;
```
Comment (trevor-vaughan) [inline build/distributions/CentOS/7/x86_64/DVD/ks/dvd/min.cfg]: Make sure this gets carried into the EL6 PR if it's not already.

File: build/distributions/CentOS/7/x86_64/yum_data/packages.yaml [hunk 5]
```diff
@@ -92,117 +83,9 @@ libevent:
...
-libselinux-static:
-  :rpm_name: libselinux-static-2.2.2-6.el7.x86_64.rpm
-  :source: http://mirror.steadfast.net/centos/7.2.1511/os/x86_64/Packages/libselinux-static-2.2.2-6.el7.x86_64.rpm
-libsepol-static:
-  :rpm_name: libsepol-static-2.1.9-3.el7.x86_64.rpm
-  :source: http://mirror.steadfast.net/centos/7.2.1511/os/x86_64/Packages/libsepol-static-2.1.9-3.el7.x86_64.rpm
 logstash:
   :rpm_name: logstash-2.3.4-1.noarch.rpm
   :source: https://download.elastic.co/logstash/logstash/packages/centos/logstash-2.3.4-1.noarch.rpm
-mcollective:
```
Comment (trevor-vaughan) [inline build/distributions/CentOS/7/x86_64/yum_data/packages.yaml]: Make sure all of these deletions get reflected in the EL6 PR

File: Puppetfile.tracking [hunk 6]
```diff
@@ -275,8 +279,8 @@ mod 'simp-oddjob',
   :git => 'https://github.com/simp/pupmod-simp-oddjob',
   :ref => 'master'
 
-mod 'simp-simp_openldap',
```
Comment (trevor-vaughan) [inline Puppetfile.tracking]: This is now simp_openldap

File: Puppetfile.tracking (hunk 1 above)
Comment (jeannegreulich) [inline Puppetfile.tracking]: I thought I had taken all of those out. I will fix it.

File: Puppetfile.tracking [hunk 7]
```diff
@@ -372,8 +378,8 @@ mod 'simp-sssd',
   :ref => 'master'
 
 mod 'simp-stunnel',
-  :git => 'https://github.com/simp/pupmod-simp-stunnel',
-  :ref => 'master'
+  :git => 'https://github.com/jeefberkey/pupmod-simp-stunnel',
```
Comment (nick-markowski) [inline Puppetfile.tracking]: should be simp
